packages/contracts-bridge/contracts/BridgeMessage.sol

```diff
@@ -37,7 +38,7 @@ library BridgeMessage {
 
     uint256 private constant TOKEN_ID_LEN = 36; // 4 bytes domain + 32 bytes id
     uint256 private constant IDENTIFIER_LEN = 1;
-    uint256 private constant TRANSFER_LEN = 97; // 1 byte identifier + 32 bytes recipient + 32 bytes amount + 32 bytes detailsHash
+    uint256 private constant TRANSFER_LEN = 129; // 1 byte identifier + 32 bytes recipient + 32 bytes amount + 32 bytes detailsHash + 32 bytes externalId
```

Comment:
this will break things for `Transfer` and `FastTransfer`, need to make a new constant `CONNEXT_TRANSFER_LEN`

Reply:
even if deprecating an old message type, you need to make sure that handling for that type still works, or else in-flight messages at the time of upgrade will be borked